smart-contracts/contracts-common/concordium-contracts-common/src/types.rs

```diff
@@ -618,15 +622,17 @@ pub struct AccountPublicKeys {
 
 pub(crate) type CredentialIndex = u8;
 
-#[derive(crate::Serialize, Debug, SchemaType)]
+#[derive(crate::Serialize, Debug, SchemaType, PartialEq, Eq)]
+#[cfg_attr(feature = "derive-serde", derive(serde::Deserialize, serde::Serialize))]
```

Comment:
So the JSON representation of this will be
```
{
    "Ed25519": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa..bbbbbbbbbbbbbbbb"
}
```

Is that wanted?

I think it's preferable to have the adjacently tagged representation https://serde.rs/enum-representations.html#adjacently-tagged

Reply:
Changed it to:
```
#[cfg_attr(feature = "derive-serde", derive(serde::Deserialize, serde::Serialize),serde(tag = "SignatureSchema", content = "Signature"))]

```